### PR TITLE
fix missing version bumps

### PR DESCRIPTION
**Pull request by cdietrich**
fix missing version bumps

### CODE REVIEW

**Comment by laeubi** [comment]
@cdietrich this are just the Integrationtests, where a default is set but the actual value is overriden later on always with the current version. We can still update them without a problem to the 3.0.

**Comment by cdietrich** [comment]
@laeubi why is build red then?

**Comment by laeubi** [comment]
Indeed the 

- [org.eclipse.tycho.test.reactor.makeBehaviour.MavenReactorMakeOptionsTest.testAlsoMake](https://ci.eclipse.org/tycho/job/tycho-github/job/master/247/testReport/junit/org.eclipse.tycho.test.reactor.makeBehaviour/MavenReactorMakeOptionsTest/testAlsoMake/)
- [org.eclipse.tycho.test.reactor.makeBehaviour.MavenReactorMakeOptionsTest.testBoth](https://ci.eclipse.org/tycho/job/tycho-github/job/master/247/testReport/junit/org.eclipse.tycho.test.reactor.makeBehaviour/MavenReactorMakeOptionsTest/testBoth/)
- [org.eclipse.tycho.test.reactor.makeBehaviour.MavenReactorMakeOptionsTest.testAlsoMakeDependentsNeedsToPickUpDependenciesOfDependents](https://ci.eclipse.org/tycho/job/tycho-github/job/master/247/testReport/junit/org.eclipse.tycho.test.reactor.makeBehaviour/MavenReactorMakeOptionsTest/testAlsoMakeDependentsNeedsToPickUpDependenciesOfDependents/)
- [org.eclipse.tycho.test.reactor.makeBehaviour.MavenReactorMakeOptionsTest.testAlsoMakeWithIndirectDependencies](https://ci.eclipse.org/tycho/job/tycho-github/job/master/247/testReport/junit/org.eclipse.tycho.test.reactor.makeBehaviour/MavenReactorMakeOptionsTest/testAlsoMakeWithIndirectDependencies/)

require a manual version bump in the `extensions.xml` due to

- https://github.com/apache/maven/pull/665
- https://github.com/apache/maven/pull/673

[org.eclipse.tycho.test.compiler.FragmentsTest.testFragment](https://ci.eclipse.org/tycho/job/tycho-github/job/master/247/testReport/junit/org.eclipse.tycho.test.compiler/FragmentsTest/testFragment/)

fails because it uses `<version>${tycho.version}</version>` instead of `<version>${tycho-version}</version>` do you like to include both fixes in your PR?

**Comment by laeubi** [comment]
Beside that I think we need to include the branch name in the cache for the `maven.yml `file as it seems to not fail due to the previously cached builds.

**Comment by cdietrich** [comment]
@laeubi i also updated `tycho-its/projects/compiler.exclude/pom.xml` let me know if it is ok

File: tycho-its/projects/compiler.exclude/pom.xml [hunk 1]
```diff
@@ -12,7 +12,7 @@
 			<plugin>
 				<groupId>org.eclipse.tycho</groupId>
 				<artifactId>tycho-maven-plugin</artifactId>
-				<version>${tycho.version}</version>
+				<version>${tycho-version}</version>
```
Comment (laeubi) [inline tycho-its/projects/compiler.exclude/pom.xml:15]: Just wondering: Is this test actually executed in the CI?

**Review by laeubi — APPROVED** [review]
Thanks looks good to me.

File: tycho-its/projects/compiler.exclude/pom.xml (hunk 1 above)
Comment (cdietrich) [reply tycho-its/projects/compiler.exclude/pom.xml:15]: good question `tycho-its/src/test/java/org/eclipse/tycho/test/compiler/CompilerExclude.java`
do we match test name patterns?

Comment (laeubi) [reply tycho-its/projects/compiler.exclude/pom.xml:15]: `CompilerExclude` must be `CompilerExcludeTest` to be executed.

Comment (cdietrich) [reply tycho-its/projects/compiler.exclude/pom.xml:15]: will do a follow up pr

Comment (laeubi) [reply tycho-its/projects/compiler.exclude/pom.xml:15]: If you like you can also include it here.

Comment (cdietrich) [reply tycho-its/projects/compiler.exclude/pom.xml:15]: done. lets see what jenkins says

Comment (laeubi) [reply tycho-its/projects/compiler.exclude/pom.xml:15]: The test fails, I think we have two options of your choice:

1. Take a look why it fails
2. Add an Ignore and create an issue to investigate the failure later on

Comment (cdietrich) [reply tycho-its/projects/compiler.exclude/pom.xml:28]: @laeubi i fixed the test. do we know when the license check will work again?

**Comment by laeubi** [comment]
Thanks for the fix, about license check I have no idea it is only "low priority service" ... So would require a strategic member complaining about or something alike...

**Pull request merged**